src/entities/SkyBox.js

```diff
@@ -11,20 +11,21 @@ class SkyBox extends Entity {
         // or you can store them in a specified path and just
         // load them up solely by filename in that path.
         
-        this.texturesList = this.getAttribute('textures');
+        this.texturesList = this.getAttribute('src');
         if (!this.texturesList) {
             return;
         }
 
         const textureLoader = THREE.CubeTextureLoader();
         const textureNames = texturesList.split(',');
-        let texture = null;
 
         let path = this.getAttribute('pathToTextures');
         const texture = !path ? textureLoader.load(textureNames) : textureLoader.load(textureNames.map((name) => path + name));
 
         // scene.background
         this.object3D.background = texture;
+        // css background
+        this.compStyle.background = texture;
```

Comment:
ah so you wouldn't set compStyle.background really ever. that's done with css. Rather I'd expect you'd use `this.compStyle.background` as an alternative to `src`.

something like:

```
this.texturesList = this.compStyle.backgroundImages
...
const textureNames = texturesList.split([some regex to parse list of paths]);
```

the difference between `src` and `background-image` would be the list of paths. each path in background-image is wrapped with `url("...")` so you'd need to do some regex.

`background-image: url("img_tree.gif"), url("paper.gif");`

Reply:
Long term it would be sick to use every feature of the `background` CSS property (color, image, repeat, position etc)  to define textures.

`background: lightblue url("img_tree.gif") no-repeat fixed center;`